src/SignalR/clients/ts/signalr/package.json

```diff
@@ -57,6 +57,7 @@
     "ws": "^6.0.0"
   },
   "resolutions": {
-    "url-parse": ">=1.5.8"
+    "//": "Avoid version that drops support for TypeScript < 3.7",
+    "@types/node": "<12.20.21"
```

Comment:
With this and removing `"@types/node": "^10.9.4"` under `devDependencies` above, `12.20.22` gets resolved everywhere. Is that better or worse than current settings, which resolve `10.17.60` for everything except `request` (which gets `12.20.20`)❔